commit dbb3a536ed9873ffa0870321f6873e564c6a9da8
Author: Jia Tan <jiat0218@gmail.com>
Date:   Wed Jun 7 00:18:30 2023 +0800

    CI: Add apt update command before installing dependencies.
    
    Without the extra command, all of the CI tests were automatically
    failing because the Ubuntu servers could not be reached properly.

diff --git a/.github/workflows/ci.yml b/.github/workflows/ci.yml
index 397a48d1..f08becb9 100644
--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -36,7 +36,9 @@ jobs:
       # Install Autotools on Linux
       - name: Install Dependencies
         if: ${{ matrix.os == 'ubuntu-latest' && matrix.build_system == 'autotools' }}
-        run: sudo apt-get install -y autoconf automake build-essential po4a autopoint gcc-multilib doxygen
+        run: |
+            sudo apt-get update
+            sudo apt-get install -y autoconf automake build-essential po4a autopoint gcc-multilib doxygen
 
       # Install Autotools on Mac
       - name: Install Dependencies
@@ -46,7 +48,9 @@ jobs:
       # Install CMake on Linux
       - name: Install Dependencies
         if: ${{ matrix.os == 'ubuntu-latest' && matrix.build_system == 'cmake' }}
-        run: sudo apt-get install -y build-essential cmake
+        run: |
+            sudo apt-get update
+            sudo apt-get install -y build-essential cmake
 
       # Install CMake on Mac
       - name: Install Dependencies